src/tests/ftest/daos_test/dfuse.py

```diff
@@ -94,7 +94,7 @@ def run_test(self, il_lib=None):
 
         command = [self.daos_test, '--test-dir', mount_dir, '--io', '--stream', '--mmap']
         if use_dfuse:
-            command.append('--lowfd')
+            command.append('--lowfd --directory')
```

Comment:
I'm slightly surprised this works.  The options should be seperate.
```suggestion
            command.extend(['--lowfd', '--directory'])
```

Reply:
I could fix this in another PR. Thank you!